src/main/java/model/process/analysis/operations/comparisons/LagSequentialAnalysis.java

```diff
@@ -1,21 +1,233 @@
 package model.process.analysis.operations.comparisons;
 
+import java.util.ArrayList;
+import java.util.Calendar;
+import java.util.Collections;
+import java.util.List;
+
+import model.data.DataRow;
+import model.data.DataTable;
+import model.data.DataTableBuilder;
+import model.data.Table;
+import model.data.describer.DataDescriber;
+import model.data.value.DateTimeValue;
+import model.exceptions.InputMismatchException;
 import model.process.analysis.operations.Event;
 
 /**
- * This class will determine a relation between events.
- * Relation will be shown by chronologically sorting events.
+ * This class will determine a relation between events. Relation will be shown
+ * by chronologically sorting events.
  * 
  * @author Louis Gosschalk 26-05-2015
  */
 public class LagSequentialAnalysis {
-	
+
+	private DataTable tableA;
+	private DataTable tableB;
+	private DataTable result;
+
+	private DataDescriber<DateTimeValue> dateA;
+	private DataDescriber<DateTimeValue> dateB;
+
+	private int positionA;
+	private int positionB;
+
+	private DateTimeValue compareA;
+	private DateTimeValue compareB;
+
+	private DataTableBuilder tableC;
+
+	private List<String> order;
+
 	/**
-	 * This class will put the events (tables) in one table, sorted chronologically.
+	 * This class constructs the LSA.
+	 * 
 	 * @param eventA
+	 *            The first event
+	 * @param dateA
+	 *            The column of the dateValues (first event)
 	 * @param eventB
+	 *            The second event
+	 * @param dateB
+	 *            The column of the dateValues (second event)
+	 * @return
+	 */
+	public LagSequentialAnalysis(Event eventA,
+			DataDescriber<DateTimeValue> dateA, Event eventB,
+			DataDescriber<DateTimeValue> dateB) {
+		tableA = checkTable(eventA.create());
+		tableB = checkTable(eventB.create());
+		this.dateA = dateA;
+		this.dateB = dateB;
+		this.tableC = new DataTableBuilder();
+
+		if (tableA.getRowCount() == 0 || tableB.getRowCount() == 0) {
+			throw new InputMismatchException("Empty event input.");
+		}
+
+		tableA = sortTable(tableA);
```

Comment:
kun je niet gebruik maken van een comparator? (http://stackoverflow.com/questions/2784514/sort-arraylist-of-custom-objects-by-property). Dat je bijvoorbeeld aan DataTable een sort functie toevoegt die zijn eigen rijen sorteert. Zo'n functie gaan we toch nodig hebben. 

Ik kijk net in de sprint plan en ik zie dat jij later in de week een sort process moet maken. Dus van mij hoef je niet nu al deze sorts aan te passen, maar plaats er dan wel een TODO comment bij en pas het aan als je de sort functies gaat maken.